cardform/src/test/java/com/mercadolibre/android/cardform/domain/FinishInscriptionUseCaseTest.kt

```diff
@@ -0,0 +1,43 @@
+package com.mercadolibre.android.cardform.domain
+
+import com.mercadolibre.android.cardform.TestContextProvider
+import com.mercadolibre.android.cardform.data.mapper.FinishInscriptionBodyMapper
+import com.mercadolibre.android.cardform.data.model.request.FinishInscriptionParam
+import com.mercadolibre.android.cardform.data.repository.FinishInscriptionRepositoryImpl
+import com.mercadolibre.android.cardform.data.service.FinishInscriptionService
+import io.mockk.mockk
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+
+internal class FinishInscriptionUseCaseTest {
+
+    @Nested
+    @DisplayName("Given a user get completion record data")
```

Comment:
I think that we can use other description for this test instead "..get completion record data ..." for example. What's your opinion?